csrankings-d.csv

```diff
@@ -268,7 +268,6 @@ Daniel Keren,University of Haifa,https://cs.haifa.ac.il/~dkeren,NOSCHOLARPAGE
 Daniel Kifer,Pennsylvania State University,http://www.cse.psu.edu/~duk17,7PQN6r4AAAAJ
 Daniel Král,Masaryk University,https://www.muni.cz/en/people/44742,crv1hRwAAAAJ
 Daniel Král',Masaryk University,https://www.muni.cz/en/people/44742,crv1hRwAAAAJ
-Daniel Kudenko,University of York,https://www-users.cs.york.ac.uk/~kudenko,uUXUMukAAAAJ

```

Comment:
Web page is not up.

Reply:
I have deleted the entry because he no longer works at York, hence his web page at York is no longer up. So far as I know, he no longer works in academia so likely does not have a new academic web page.